mmdet/models/dense_heads/base_dense_head.py

```diff
@@ -57,3 +57,22 @@ def forward_train(self,
         else:
             proposal_list = self.get_bboxes(*outs, img_metas, cfg=proposal_cfg)
             return losses, proposal_list
+
+    def simple_test(self, feats, img_metas, rescale=False, postprocess=True):
+        """Test function without test-time augmentation.
+
+        Args:
+            feats (tuple[torch.Tensor]): Multi-level features from the
+                upstream network, each is a 4D-tensor.
+            img_metas (list[dict]): List of image information.
+            rescale (bool, optional): Whether to rescale the results.
+                Defaults to False.
+            postprocess (bool, optional): Whether to perform post-processing
+                by bbox2result. Defaults to True.
+        Returns:
+            list[list[np.ndarray]]: BBox results of each image and classes.
+                The outer list corresponds to each image. The inner list
+                corresponds to each class.
+        """
+        return self.simple_test_bboxes(
```

Comment:
Seems you add this arguments `postprocess` to get raw results of boxes in `yolact`, but I believe original design keep `bbbox2results` is single-stage would a better design and can do the same thing, Is there other reasons?

In addition, I think keeping this operation in the `SingleStageDetector` instead of `simple_test_bboxes` would  be more consistent with the two-tage models.
just like
https://github.com/open-mmlab/mmdetection/blob/24b6f9319f7657336950fde9b5ab562201d90963/mmdet/models/roi_heads/standard_roi_head.py#L233

Reply:
I added the argument `postprocess` to simplify code for onnx.
https://github.com/open-mmlab/mmdetection/blob/7578b6c13b54f7828064c7b5deb7e42a4318f731/mmdet/models/detectors/single_stage.py#L117-L118
The original reason disappeared after https://github.com/open-mmlab/mmdetection/pull/5205

Reply:
The overall design looks good to me, and I would help to move `bbbox2results` to `simple_test` of single_stage.py, It would be merged soon, Thanks for your contribution.